main.cpp

```diff
@@ -1,12 +1,17 @@
 #include "fibonancci.h"
-#include <iostream>
+#include <iostream.h>
 using namespace std;
 
-int main (){
+int main ()
+{
     Fibonancci fib; 
-    int total=0;
-    for (int i = 0; i < 20; i++)
-        total += fib.produce_output(i);
-    cout << total << endl;
+    int term=0,n;
```

Comment:
Nice one for raising a PR! Could we name `n` something like `userInput` 😄

Reply:
it depends upon u

Reply:
:)